Changelog.md

```diff
@@ -2,11 +2,67 @@
 This page defines all the changes that were done in the API. There is a difference between the version of the API and the version of iProva. A new major API version will only be created when we have to introduce breaking changes to the API. New routes, enhanced routes and bug fixes can be introduced with a new iProva version without creating a new API version.
 
 
+## Changes in iProvba 5.15
+
+### General changes
+The base path was changed from /iProva to /iProva/API and the /api/ prefix is removed from all routes. This doesn't change anything for consumers.
+
+Setting routes are now more specific for the setting you want to get e.g
```

Comment:
Only makes it less clear for the consumer, lets just remove it because it does not impact them